### PR TITLE
Added the folder with homework.

### PR DESCRIPTION
**Pull request by MarianaKhl**

### CODE REVIEW

File: .idea/.gitignore
Comment (edanilovets) [inline .idea/.gitignore:1]: .idea/ contains personal developers settings, this folder should not be a part of repo
add it to .gitignore

File: .DS_Store
Comment (edanilovets) [inline .DS_Store:1]: .DS-Store it is internal OS file, it should not be a part of repo
add it to .gitignore

File: lesson_01/homework_01.py
Comment (edanilovets) [inline lesson_01/homework_01.py:1]: why are you deleting homework_01 in this PR? let is stay

**Comment by edanilovets** [comment]
Usually we don't add file names to the PRs title, because there can be many of them in one PR. Title should look pretty and informative. Please update it.

File: lesson_01/homework_01.py
Comment (MarianaKhl) [reply lesson_01/homework_01.py:1]: I accidentally added lesson_01 to the homework03 branch, and if I leave lesson_01 in this PR, will I add two folders with lesson_01 to the main branch?

Comment (edanilovets) [reply lesson_01/homework_01.py:1]: @MarianaKhl in your PR you are deleting this folder and file homework_01.py. If you merge it this file will be deleted from the main branch.

Comment (edanilovets) [reply lesson_01/homework_01.py:1]: It is not accidentally, when you create the branch from main, it must have lesson1, it is ok.

Comment (MarianaKhl) [reply lesson_01/homework_01.py:1]: Thanks for the tips! Added changes and returned the folder with lesson_01.

**Comment by MarianaKhl** [comment]
Hi! Can you recheck my homework:
https://github.com/MarianaKhl/Lesson_2-GitHub-/pull/2

**Review by edanilovets — COMMENTED** [review]
@MarianaKhl You haven't removed .idea/, it is still going to be in the repo after the merge
I can show how to do it on the lesson if its ok?

**Comment by MarianaKhl** [comment]
Yes, it's Ok.

Thanks!

**Comment by MarianaKhl** [comment]
Hi again!

Please check the recycled homework. Created a new Pull Request. Sorry for
the one Ukrainian commit; this is an old commit.

https://github.com/MarianaKhl/Lesson_2-GitHub-/pull/3

**Pull request closed without merging**